### PR TITLE
Integration/homepage

### PR DESCRIPTION
**Pull request by nicolekurtz93**
Added functionality for bar chart when user clicks on state that shows total number of gun deaths per year. 

Create About page skeleton.

### CODE REVIEW

File: gun-violence-dashboard/src/pages/Homepage.js [hunk 1]
```diff
@@ -139,6 +187,7 @@ function Homepage() {
         <>
             <div className="m-3">
                 <h1 className="text-center">Number of deaths due to firearms per 100,000 population in the US</h1>
+                <h2 className="text-center">Click on state to learn more</h2>
```
Comment (stevensonmat2) [inline gun-violence-dashboard/src/pages/Homepage.js:191]: Styling of data could be improved here I think. Text alignment is off, and the stats could pop more.

File: gun-violence-dashboard/src/pages/About.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
```
Comment (stevensonmat2) [inline gun-violence-dashboard/src/pages/About.js:1]: I think the max width for the text should be more narrow; on wider screens it stretches out a lot is harder to read.

File: gun-violence-dashboard/src/pages/Homepage.js [hunk 3]
```diff
@@ -1,22 +1,55 @@
 import React, { useEffect, useState } from "react";
 import USAMap from "react-usa-map";
 import axios from 'axios';
+import {
```
Comment (stevensonmat2) [inline gun-violence-dashboard/src/pages/Homepage.js:4]: a general note, I think the loading icon should remain until all of the data is loaded; there is still some jumping around as data loads after the load icon disappears

**Review by stevensonmat2 — CHANGES_REQUESTED** [review]

File: gun-violence-dashboard/src/pages/About.js (hunk 2 above)
Comment (nicolekurtz93) [reply gun-violence-dashboard/src/pages/About.js:1]: I can do that!

File: gun-violence-dashboard/src/pages/Homepage.js (hunk 3 above)
Comment (nicolekurtz93) [reply gun-violence-dashboard/src/pages/Homepage.js:4]: Do you have some suggestions on how to resolve this? I am awaiting all the data, but it still jumps.

**Review by stevensonmat2 — APPROVED** [review]

**Pull request merged**